web_widget_domain_v11/static/src/copied-js/domain_selector_dialog.js

```diff
@@ -0,0 +1,47 @@
+odoo.define("web.DomainSelectorDialog", function (require) {
```

Comment:
There are thousands of JS lines needed. I just copy-pasted all those involved and made sure there was no module name conflict, but honestly I'd prefer not to have to look deeply into each one of them since the life span of this module is going to be so short (just 1 version).